lib/lte_link_control/lte_lc.c

```diff
@@ -690,9 +699,12 @@ static int connect_lte(bool blocking)
 			LOG_INF("Network connection attempt timed out");
 
 			if (IS_ENABLED(CONFIG_LTE_NETWORK_USE_FALLBACK) &&
-			    (sys_mode_target == sys_mode_preferred)) {
-				sys_mode_target = sys_mode_fallback;
-				retry = true;
+			    (tries > 0) ) {
+				if(sys_mode_target == sys_mode_preferred) {
```

Comment:
Also here there should be a space after `if`.

Reply:
Addressed!